packages/bpk-component-bottom-sheet/src/BpkBottomSheet.module.scss

```diff
@@ -17,16 +17,156 @@
  */
 
 @use '../../unstable__bpk-mixins/tokens';
+@use '../../unstable__bpk-mixins/borders';
 @use '../../unstable__bpk-mixins/breakpoints';
+@use '../../unstable__bpk-mixins/radii';
+@use '../../unstable__bpk-mixins/shadows';
+@use '../../unstable__bpk-mixins/typography';
+@use '../../unstable__bpk-mixins/utils';
+
+$close-button-width: tokens.bpk-spacing-lg() * 2;
 
 .bpk-bottom-sheet {
-  &--container {
+  z-index: tokens.$bpk-zindex-modal;
+  width: 100%;
+  max-width: tokens.$bpk-modal-max-width;
+  margin: auto;
+  transform: scale(1);
+  transition:
+    opacity tokens.$bpk-duration-sm ease-in-out,
+    transform tokens.$bpk-duration-sm ease-in-out;
+  outline: 0;
+  background-color: tokens.$bpk-modal-background-color;
+  opacity: tokens.$bpk-modal-opacity;
+
+  // required to prevent the header extending beyond the rounded corners
+  overflow: hidden;
+  overflow-y: scroll;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+
+  // FIX: Prevents ugly flickering when tapping inside the modal in Mobile Safari
+  -webkit-tap-highlight-color: transparent;
+
+  @include shadows.bpk-box-shadow-xl;
+  @include radii.bpk-border-radius-sm;
+
+  @include breakpoints.bpk-breakpoint-mobile {
+    position: fixed;
+    bottom: 0;
+    height: fit-content;
+    max-height: 90%;
+    margin-bottom: 0;
+    border-radius: tokens.$bpk-border-radius-lg tokens.$bpk-border-radius-lg 0 0;
+    overflow-x: hidden;
+  }
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+
+  &--appear {
+    animation-duration: 0.24s;
```

Comment:
Can we/should we be using Backpack standard animation values here? 

https://www.skyscanner.design/latest/foundations/animation/overview-DTJrpm76

Otherwise it calls into question what are these 'random' numbers mean